src/store/index.jsx

```diff
@@ -1,7 +1,35 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { configureStore } from "@reduxjs/toolkit";
+import { persistReducer, persistStore } from 'redux-persist';
+import { authSlice } from "./slices/auth.slice";
+import { userSlice } from "./slices/user.slice";
+import { homesApi } from './apis/hotels/getPopularHotels.endpoint'; // Importing homesApi
+import { combineReducers } from 'redux';
+import storage from 'redux-persist/lib/storage';
+import {hotelsApi} from './apis/hotels/hotels.api';
```

Comment:
Please run `npx eslint --fix` before committing your final changes
```suggestion
import { hotelsApi } from './apis/hotels/hotels.api';
```